docs/README.md

```diff
@@ -1,20 +1,22 @@
 # GEMD Documentation
 
-GEMD stands for Generalized Expression of Materials Data. It's an open source format initially developed by the fine folks at Citrine Informatics. 
+GEMD stands for Generalized Expression of Materials Data. It's an open source format initially developed by the fine folks at Citrine Informatics.
+
+![Material Graph](img/FORMULATIONS_ILLS_2.png)
```

Comment:
I think this figure could do a better job of distinguishing between ingredient names and material names from the get-go, which is a consistent point of confusion.

Reply:
Agree, see comment above.